documents/openapi/openapi.yaml

```diff
@@ -2169,6 +2169,8 @@ components:
           type: string
           format: date-time
           nullable: true
+        cost_in_millicents:
+          type: number
```

Comment:
This isn't being displayed in the swagger editor for the GET /notifications sms example 200 response. Not sure that this is in the correct location in the file.

Reply:
OK, this is showing in the schema, but not in the example, in swagger. Please update the example (starts around line 1164) so that it shows up in the sms example.
![Screen Shot 2023-01-20 at 1 08 18 PM](https://user-images.githubusercontent.com/111803622/213796420-a6695120-25c1-4094-a77c-a33add821859.jpg)

Reply:
I added the new columns to the "sms" and "email" example in both spec files.